source/agora/node/Node.d

```diff
@@ -338,9 +347,70 @@ public class Node : API
         }
     }
 
+    /***************************************************************************
+
+        Returns an instance of a EnrollmentManager
+
+        Params:
+            data_dir = path to the data dirctory
+            node_config = the node config
+
+        Returns:
+            the enrollment manager
+
+    ***************************************************************************/
+
+    protected EnrollmentManager getEnrollmentManager (string data_dir,
+        in NodeConfig node_config)
+    {
+        return new EnrollmentManager(buildPath(data_dir, "validator_set.dat"),
+            node_config.key_pair);
+    }
+
     /// GET: /merkle_path
     public Hash[] getMerklePath (ulong block_height, Hash hash) @safe
     {
         return this.ledger.getMerklePath(block_height, hash);
     }
+
+    /// GET: /get_utxos
+    public UTXOSetValue[Hash] getUTXOs (PublicKey address, TxType type,
```

Comment:
This method is allocating two associative arrays. We could add an optional `min_amount` parameter to  `UTXOSet.getUTXOs` and implement the amount filtering there?

Reply:
Sure, we could. the function will be enhanced in another PR.